scripts/build/termux_extract_dep_info.sh

```diff
@@ -2,15 +2,16 @@
 
 termux_extract_dep_info() {
 	( # Do everything in a subshell do avoid variable hell outside the function
-	PKG=$1
-	PKG_DIR=$2
+	PKG="$1"
+	PKG_DIR="$2"
 
 	# set TERMUX_SUBPKG_PLATFORM_INDEPENDENT to
 	# parent package's value and override if
 	# needed
-	TERMUX_PKG_PLATFORM_INDEPENDENT=false
+	TERMUX_PKG_ON_DEVICE_BUILD_NOT_SUPPORTED="false"
+	TERMUX_PKG_PLATFORM_INDEPENDENT="false"
 	TERMUX_PKG_REVISION="0"
-	source ${PKG_DIR}/build.sh
+	source "${PKG_DIR}/build.sh"
 
 	# debian version
 	VER_DEBIAN="$TERMUX_PKG_VERSION"
```

Comment:
Mixed indentation on lines 31 and 32.
![image](https://github.com/user-attachments/assets/12329f0f-7f3f-4504-a6f6-a9b89ab520fd)

Here's that entire block "fixed up", there's a couple other things like quoting and shellcheck sources that can't be followed.
```bash
	if [[ "$PKG" != "$(basename "${PKG_DIR}")" && "${PKG/-glibc/}" != "$(basename "${PKG_DIR}")" ]]; then
		if [[ "$TERMUX_INSTALL_DEPS" == "false" || \
				"$TERMUX_PKG_NO_STATICSPLIT" = "true" || \
				"${PKG/-static/}-static" != "${PKG}" ]]; then
			# shellcheck source=/dev/null
			if [[ -f "${PKG_DIR}/${PKG}.subpackage.sh" ]]; then
				source "${PKG_DIR}/${PKG}.subpackage.sh"
			else
				source "${PKG_DIR}/${PKG/-glibc/}.subpackage.sh"
			fi
		fi
	fi
```

Reply:
Fixed.